demos/projects/ESPRESSIF/esp32/config/_lwipopts.h

```diff
@@ -0,0 +1,338 @@
+/* Copyright (c) Microsoft Corporation.
```

Comment:
lwipopts.h is part of lwip component for esp32, you do not need this file plus it has _ prefixed to it

[https://github.com/espressif/esp-idf/blob/166c30e7b2ed1dcaae56179329540a862915208a/components/lwip/port/esp32/include/lwipopts.h](https://github.com/espressif/esp-idf/blob/166c30e7b2ed1dcaae56179329540a862915208a/components/lwip/port/esp32/include/lwipopts.h)

Reply:
Indeed. Removed.